update inbound filters page with Session Replay details (#14122)
getsentry/sentry-docs: docs/concepts/data-management/filtering/index.mdx

- `*/ping`
- `*/up`

## Session Replay Filtering

Inbound data filters have partial support for [Session Replay](/product/explore/session-replay/). Only a subset of the available inbound filters apply to Session Replays.

The following inbound filters **do** apply to Session Replays:

- **IP Addresses** - Filters replays based on the originating IP address
- **Releases** - Filters replays from specific release versions
- **Request URLs** - Filters replays based on the URL where the replay was captured
- **User-Agents** - Filters replays based on the browser's user-agent string

The following inbound filters **do not** apply to Session Replays:

- Error messages
- Browser extension errors  
- Web crawler errors
- Legacy browser filters
- React hydration errors
- ChunkLoadErrors
- Health check transactions

> **Note**: Because filtered outcomes are emitted per **segment** whereas successful outcomes are emitted per **replay** (a replay being a collection of segments), you may see a noticeable increase in filtered outcomes on your [Stats](https://sentry.io/orgredirect/organizations/:orgslug/stats) page when Session Replay filtering is active. This is expected behavior and not an error.

Reviewer: ```suggestion
**Note**: Because filtered outcomes are emitted per **segment** whereas successful outcomes are emitted per **replay** (a replay being a collection of segments), you may see a noticeable increase in filtered outcomes on your [Stats](https://sentry.io/orgredirect/organizations/:orgslug/stats) page when Session Replay filtering is active. This is intended behavior.
```
